src/main/java/io/stargate/sgv2/jsonapi/api/model/command/Command.java

```diff
@@ -27,4 +27,51 @@
   @JsonSubTypes.Type(value = GeneralCommand.class),
   @JsonSubTypes.Type(value = CollectionCommand.class),
 })
-public interface Command {}
+public interface Command {
+
+  /**
+   * commandName that refers to the public command name
+   *
+   * <p>e.g. FindKeyspacesCommand publicCommandName -> findKeyspaces. CreateCollectionCommand
+   * publicCommandName -> createCollection
+   */
+  PublicCommandName publicCommandName();
```

Comment:
nit: we can just call the enum CommandName and have Command.commandName()

on the enum can we change `publicCommandName` and it's accessor to `apiName` we have that in other places.

Reply:
Thanks!